### PR TITLE
Bug 1033917 - [Rocketbar][RTL] Vertical Homescreen E.me search input should be right aligned in RTL locales +autoland

### PR DESCRIPTION
**Pull request by anefzaoui**
This patch:
Fixes Homescreen rochetbar search RTL layout.
Fixes Rocketbar clicked state of the search RTL layout.
Fixes browser chrome controls.
Fixes Rocketbar problem of being initiated from the left corner of the status bar even when in RTL.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=58ec331db41b)

File: shared/elements/gaia_progress/script.js [hunk 1]
```diff
@@ -18,9 +18,13 @@ window.GaiaProgress = (function(win) {
 
     ComponentUtils.style.call(this, baseurl);
 
-    if (this.hasAttribute('animated')) {
-      this.start();
-    }
+    var self = this;
+    window.addEventListener('localized', function __start(evt){
```
Comment (KevinGrandon) [inline shared/elements/gaia_progress/script.js]: Please instead use an arrow function and remove the ```self``` variable.

```js
window.addEventListener('localized', evt => {
  // Use this.start/stop here.
});
```

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=669c5044f2bd)

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5b9270d75657)

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=792dc3ac0459)

**Pull request merged**